imports/ui/components/pwd-strength.js

```diff
@@ -1,42 +0,0 @@
-/* Password strength validator */
```

Comment:
This way we just load one file when a form is called. As I had other calls loaded just on this part, I decided to move it here. Is it fine?